scss/.sassdocrc

```diff
@@ -1,12 +1,12 @@
 # .sassdocrc (yaml)
 
-dest: './documentation/0.6.75'
+dest: './documentation/herman-test'
 descriptionPath: './README.md'
 shortcutIcon: '../sinister-logo.svg'
 herman:
   sass:
     includePaths:
-      - 'test-sin/scss'
-    use:
-      - file: 'sin'
-      - namespace: '*'
+      - 'sin-core'

```

Comment:
This should be the load paths used to resolve any `@use` or `@import` declarations inside `@example` annotation code.

Reply:
I have a test folder that imports the library via `npm link` . This folder is in `.gitignore` so you did not have access to it. 

It  took me several hours to grasp how this part works in combination with the `npm run doc-herman`  source. Some of my issues were definitely in this part.